project/src/App.css

```diff
@@ -0,0 +1,81 @@
+#root {
+  max-width: 1280px;
+  /* margin: 0 auto; */
+  /* padding: 2rem; */
+  text-align: center;
+}
+
+.logo {
```

Comment:
nit: these logo styles come from the Vite framework out-of-the-box - remove if no longer needed/used